### DISCUSSION TITLE
a1111 sd on runpod

### DISCUSSION THREAD

**Discussion by vesperialuna** [TheLastBen/fast-stable-diffusion#2160] [General]
hi thelastben,

i have encountered a problem when running the template on runpod. usually when i open jupyter, i will edit/add the necessary before relaunching the webui. example, i added a few models under the sd models folder and a few embeddings as well. on top of that i will edit the webui-user.sh file so that i can launch the gradio link. everything runs well. the only issue is when i try to change the model in the ui, it kills the session and relaunches it again. when everything is fully loaded, it still stuck to the first model added in the folder. i have changed a few pod and issue is the same. any ideas how to work around this?

thank you so much in advance and for all your hardwork.

**Comment by TheLastBen** [comment]
hi, why do you use webui-user.sh ? are you use fast-stable-diffusion template ? https://www.runpod.io/console/gpu-browse?template=runpod-stable-unified

**Reply by vesperialuna** [reply]
now i see my problem i have been using the wrong template omg. when i opened that link, then i saw the auto-models folder i know i have been using the wrong one before. thanks ben. issue resolved. 

edit: just to be sure at the beginning i put my read token for huggingface, right? and the last one is of course the write token?

**Reply by TheLastBen** [reply]
those for saving and backing up your sd folder, they are optional
